python/lib/client/dmod/client/request_clients.py

```diff
@@ -460,6 +531,55 @@ async def delete_dataset(self, name: str, **kwargs) -> bool:
         self.last_response = await self.async_make_request(request)
         return self.last_response is not None and self.last_response.success
 
+    async def download_item_block(self, dataset_name: str, item_name: str, blk_start: int, blk_size: int) -> AnyStr:
```

Comment:
Likewise in this method. It would helpful to return some kind of Result type indicating success or failure and it might also be nice to return the number of bytes read / downloaded since the block size is maximum size to download.